scripts/upgrade-users-sparse-index.js

```diff
@@ -0,0 +1,59 @@
+﻿'use strict';
+// Set the Node ENV
+process.env.NODE_ENV = 'development';
+
+var mongoose = require('mongoose'),
+  chalk = require('chalk'),
+  mg = require('../config/lib/mongoose');
```

Comment:
I think this should be mongoose instead of mg to be consistent with the rest of the project.